<?xml version='1.0' encoding='UTF-8'?>

<launch>

  <arg name="robot_type" default="tracked"/>
  <arg name="robot_name" default="airport_robot"/>
  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find airplane_formation_control)/config/control_config_$(arg robot_type).yaml" command="load"  ns="/$(arg robot_name)"/>

  <!-- load controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  ns="/$(arg robot_name)/airport_robot"
  args="joint_state_controller 
  rear_right_velocity_controller 
  rear_left_velocity_controller 
  front_right_velocity_controller 
  front_left_velocity_controller"/>
  <!-- velodyne_support_controller -->
  

  <node name="cmdvel2gazebo_c919" pkg="airplane_formation_control" type="cmdvel2gazebo_c919.py" respawn="true" output="screen">
      <param name="topic_name" value="$(arg robot_name)/airport_robot" />
  </node>



</launch>
